src/components/ExportDataModal.js

```diff
@@ -97,23 +98,25 @@ export default _.flow(
         content: `WARNING: ${runningSubmissionsCount} workflows are currently running in this workspace. ` +
           'Copying the following data could cause failures if a workflow is using this data.'
       }),
-      requiredFormLabel('Destination'),
-      h(WorkspaceSelector, {
-        workspaces: _.filter(Utils.isValidWsExportTarget(workspace), workspaces),
-        value: selectedWorkspaceId,
-        onChange: v => this.setState({ selectedWorkspaceId: v })
-      }),
+      !((hardConflicts.length !== 0) || moreToDelete || (softConflicts.length !== 0)) && h(Fragment, [
```

Comment:
For the `.length === 0`'s, you could actually just do `foo.length` or `!!foo.length`